src/components/combo_box/combo_box_input/_combo_box_pill.scss

```
@@ -17,6 +17,10 @@
     margin: ($euiSizeXS + 1px) $euiSizeXS 0 0;
   }
 
+  .euiComboBox--compressed &--plainText {
+    margin: $euiSizeXS $euiSizeXS 0 0;
```

Let's simplify this slightly - if the margins of the other 3 sides aren't changing, there's no need to set thme
```suggestion
    margin-top: $euiSizeXS;
```

---

😆 I almost went that direction and even asked myself "What would @cee-chen do in this situation?" I'll take your commit suggestion wholesale.